src/styles/theme/types.ts

```diff
@@ -1,11 +1,24 @@
 export type IBreakpoints = {
   breakpoints: {
-    extraSmall: string;
-    small: string;
-    medium: string;
-    large: string;
-    extraLarge: string;
-    big: string;
+    mobileS: string;
+    mobileM: string;
+    mobileL: string;
+    tablet: string;
+    laptop: string;
+    desktop: string;
+    desktopL: string;
+  };
+};
+
+export type IDevices = {
```

Comment:
O type IBreakpoints foi criado para englobar tudo o que seja relacionado as quebras de linha, logo, você pode adicionar as propriedades de IDevices nele.

Quando for usar o destructuring lá em `src/styles/theme/themes.ts` a variável "device" será adicionada normalmente e estará disponível para ser usada assim como você está usando o `theme.device` em `src/components/AddMaterialForm/style.ts`.

Lembre-se também de mesclar os valores de device em breakpoints, lá em `src/styles/theme/themes.ts`
```suggestion
```

Reply:
ah tá, entendido vou fazer aqui e já mando 🤙

Reply:
Você pode alterar o nome para `globalBreakpoints` definir os valores de breakpoints separado de devices e juntar em `globalBreakpoints`.

A desconstrução agora, será a partir de `...globalBreakpoints` no arquivo `src/styles/theme/themes.ts` linha 40 e 50.

```JavaScript
const breakpoints = {
  mobileS: "320px",
  mobileM: "375px",
  mobileL: "425px",
  tablet: "768px",
  laptop: "1024px",
  desktop: "1440px",
  desktopL: "2560px",
};

const devices = {
  mobileS: `(min-width: ${breakpoints.breakpoints.mobileS})`,
  mobileM: `(min-width: ${breakpoints.breakpoints.mobileM})`,
  mobileL: `(min-width: ${breakpoints.breakpoints.mobileL})`,
  tablet: `(min-width: ${breakpoints.breakpoints.tablet})`,
  laptop: `(min-width: ${breakpoints.breakpoints.laptop})`,
  desktop: `(min-width: ${breakpoints.breakpoints.desktop})`,
  desktopL: `(min-width: ${breakpoints.breakpoints.desktopL})`,
};

const globalBreakpoints: IBreakpoints = {
  breakpoints,
  devices,
};
```

Reply:
realizei as alterações